src/main/java/github/scarsz/discordsrv/util/LangUtil.java

```diff
@@ -196,6 +196,97 @@
                     "6. 重啟伺服器\n" +
                     "\n" +
                     "\n");
+        }}), RESPECT_CHAT_PLUGINS_DISABLED(new HashMap<Language, String>() {{
+            put(Language.EN,
+                    "\n" +
+                    "\n" +
+                    "RespectChatPlugins is disabled, this option is for TESTING PURPOSES ONLY\n" +
+                    "and should NEVER be disabled on production servers.\n" +
+                    "Disabling the option will cause cancelled messages to be forwarded to Discord\n" +
+                    "including but not limited to private messages or staff chat messages without /commands\n" +
+                    "\n" +
+                    "\n");
+            put(Language.FR,
+                    "\n" +
+                    "\n" +
+                    "RespectChatPlugins est désactivé, cette option est UNIQUEMENT À DES FINS DE TEST\n" +
+                    "et ne doit JAMAIS être désactivé sur les serveurs de production.\n" +
+                    "La désactivation de cette option entraînera le transfert des messages annulés vers Discord\n" +
+                    "y compris, mais sans s'y limiter, les messages privés ou les messages de discussion du personnel sans commandes\n" +
+                    "\n" +
+                    "\n");
+            put(Language.DE,
+                    "\n" +
+                    "\n" +
+                    "RespectChatPlugins ist deaktiviert. Diese Option dient nur zum Testen von Zwecken\n" +
+                    "und sollte NIEMALS auf Produktionsservern deaktiviert werden.\n" +
+                    "Durch Deaktivieren der Option werden abgebrochene Nachrichten an Discord weitergeleitet\n" +
+                    "einschließlich, aber nicht beschränkt auf private Nachrichten oder Chat-Nachrichten von Mitarbeitern ohne / Befehle\n" +
+                    "\n" +
+                    "\n");
+            put(Language.JA,
+                    "\n" +
+                    "\n" +
+                    "RespectChatPluginsは無効になっています。このオプションは、目的をテストするためだけのものです\n" +
+                    "実稼働サーバーでは無効にしないでください。\n" +
+                    "オプションを無効にすると、キャンセルされたメッセージがDiscordに転送されます\n" +
+                    "/commandsを使用しないプライベートメッセージまたはスタッフチャットメッセージが含まれますが、これらに限定されません\n" +
+                    "\n" +
+                    "\n");
+            put(Language.KO,
+                    "\n" +
+                    "\n" +
+                    "RespectChatPlugins가 비활성화되었습니다.이 옵션은 테스트 목적으로 만 사용됩니다.\n" +
+                    "프로덕션 서버에서는 절대 비활성화하지 않아야합니다.\n" +
+                    "이 옵션을 비활성화하면 취소 된 메시지가 불일치로 전달됩니다.\n" +
+                    "/command가없는 개인 메시지 또는 직원 채팅 메시지를 포함하지만 이에 국한되지는 않습니다.\n" +
+                    "\n" +
+                    "\n");
+            put(Language.NL,
+                    "\n" +
+                    "\n" +
+                    "RespectChatPlugins is uitgeschakeld, deze optie is ALLEEN voor TESTEN VAN DOELEINDEN\n" +
```

Comment:
Make this `RespectChatPlugins is uitgeschakeld, deze optie is ALLEEN voor TESTEN VAN DOELEINDEN\n` into `RespectChatPlugins is uitgeschakeld, deze optie is ALLEEN voor TEST DOELEINDEN\n`

Reply:
Thanks, pro tip there's the "Insert a suggestion" button, the + and - on top of each other